tools/ci/travis/travis-ci.sh

```diff
@@ -37,13 +37,20 @@ else
   FETCH_HEAD=$TRAVIS_COMMIT
 fi
 
-if [[ $TRAVIS_COMMIT != $(git rev-parse FETCH_HEAD) ]] ; then
-  echo "WARNING! Travis TRAVIS_COMMIT and FETCH_HEAD do not agree!"
-  if git cat-file -e $TRAVIS_COMMIT 2> /dev/null ; then
-    echo "TRAVIS_COMMIT exists, so going with it"
-  else
-    echo "TRAVIS_COMMIT does not exist; setting to FETCH_HEAD"
-    TRAVIS_COMMIT=$FETCH_HEAD
+if [[ $TRAVIS_EVENT_TYPE = 'push' ]] ; then
```

Comment:
There's no FTECH_HEAD for a push, so this just ensure that `TRAVIS_COMMIT` does exist in the repo. Prior to this change, there was an error message from git complaining about the lack of the `FETCH_HEAD` followed by the "TRAVIS_COMMIT exists, so going with it" message.